Carmunication/Controllers/SessionStartViewController.swift

```diff
@@ -7,10 +7,138 @@
+        btn.layer.cornerRadius = 8
+        return btn
+    }()
+    // 상단 그룹에 대한 컬렉션뷰입니다.
+    fileprivate let groupCollectionView: UICollectionView = {
+        let layout = UICollectionViewFlowLayout()
+        layout.scrollDirection = .horizontal    // 좌우로 스크롤
+        layout.sectionInset = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 20) // 여백 조정
 
+        let collectionView = UICollectionView(frame: .zero, collectionViewLayout: layout)
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.register(GroupCollectionViewCell.self, forCellWithReuseIdentifier: "groupCell")
+        collectionView.showsHorizontalScrollIndicator = false   // 스크롤바 숨기기
+        return collectionView
+    }()
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .systemBackground
+        setCollectionView()
+        setJourneyTogetherButton()
+    }
+}
+
+extension SessionStartViewController {
+    private func setCollectionView() {
+        view.addSubview(groupCollectionView)
+        groupCollectionView.backgroundColor = .white
+        NSLayoutConstraint.activate([
+            groupCollectionView.topAnchor.constraint(equalTo: view.topAnchor, constant: 20),
+            groupCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0),
+            groupCollectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0),
+            groupCollectionView.heightAnchor.constraint(equalTo: groupCollectionView.widthAnchor, multiplier: 0.5)
+        ])
+        groupCollectionView.delegate = self
+        groupCollectionView.dataSource = self
+    }
+    private func setJourneyTogetherButton() {
+        view.addSubview(journeyTogetherButton)
+        NSLayoutConstraint.activate([
+            journeyTogetherButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -120),
+            journeyTogetherButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            journeyTogetherButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            journeyTogetherButton.heightAnchor.constraint(equalToConstant: 120)  // 버튼의 높이 조절
+        ])
+        journeyTogetherButton.addTarget(self, action: #selector(presentModalQueue), for: .touchUpInside)
+    }
+    @objc private func presentModalQueue() {
+        let modalQueueViewController = ModalQueueViewController()
+        self.present(modalQueueViewController, animated: true)
+    }
+}
+
+extension SessionStartViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return groupData.count
+    }
+    func collectionView(_ collectionView: UICollectionView,
+                    layout collectionViewLayout: UICollectionViewLayout,
+                    sizeForItemAt indexPath: IndexPath) -> CGSize {
```

Comment:
SwiftLint에서 경고 메시지 출력되고 있는데 확인해주세요 !
```suggestion
    func collectionView(
        _ collectionView: UICollectionView,
        layout collectionViewLayout: UICollectionViewLayout,
        sizeForItemAt indexPath: IndexPath
    ) -> CGSize {
```

Reply:
아아 넵 수정하겠슴다!